install/configure_administrative_access.md

```diff
@@ -51,7 +60,7 @@ as protections for FAS and CAPMC actions.
 
 See [Set BMC `Management` Role](../operations/hardware_state_manager/Set_BMC_Management_Role.md).
 
-## 4. Lock management nodes
+## 3. Lock management nodes
```

Comment:
Looks like the table of contents needs to be updated for several of these since this PR reindexed the headers.

Reply:
Fixed.